builtin/providers/azurerm/config.go

```diff
@@ -0,0 +1,167 @@
+package azurerm
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure"
```

Comment:
Ah we specifically depend on "the go-autorest used by the Azure SDK" - fair play.